docs/reference/ml/anomaly-detection/ml-configuring-alerts.asciidoc

```diff
@@ -0,0 +1,85 @@
+[role="xpack"]
+[[ml-configuring-alerts]]
+= Configuring {anomaly-detect} alerts
+
+{anomaly-detect-cap} alerts run scheduled checks on an {anomaly-job} or a group 
+of jobs to detect anomalies with certain conditions. If an {anomaly} meets the 
```

Comment:
There's no shared attribute in https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc that matches this:
```suggestion
of jobs to detect anomalies with certain conditions. If an anomaly meets the 
```